### DISCUSSION TITLE
[Feature] browse duplicate images

### DISCUSSION THREAD

**Discussion by robkooper** [immich-app/immich#10576] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When looking at the duplicates it shows you how many duplicates there are, and it shows the first duplicate. It does however not allow me to go to the next/prev duplicate, I have to make an action before I can do something.

Sometimes I know there is an duplicate (esp with external library) and will have to fix that on the drive itself since it is mounted as read-only, but there is no way to go the next duplicate.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#10402
